data/human/free worlds checkmate.txt

```diff
@@ -931,7 +931,7 @@ mission "FWC Checkmate 1C"
 				`	"Understood. We will depart immediately."`
 					accept
 				`	"Are we really sure we want to attack Earth?"`
-			`	"Have they given us any other choice?" asks Tomek. "We tried diplomacy. And we cannot hope to defeat the entire Navy. So, unless we wish to surrender and all be put on trial as war criminals, we need to press on and gain the upper hand in the only way we can. The Senate and the Council are agreed on this." You have no choice but to follow his orders and to begin getting your fleet ready for the next attack.`
+			`	"Have they given us any other choice?" asks Tomek. "We tried diplomacy. And we cannot hope to defeat the entire Navy. So, unless we wish to surrender and all be put on trial as war criminals, we need to press on and gain the upper hand in the only way we can. The Senate and the Council are in agreement on this." You have no choice but to follow his orders and to begin getting your fleet ready for the next attack.`
```

Comment:
I don't know if this is *good* English, but it is definitely reasonable conversational dialog to say that x and y are agreed on something.